core/src/main/java/io/grpc/internal/InternalServer.java

```diff
@@ -47,4 +50,10 @@
    * available or does not make sense.
    */
   int getPort();
+
+  /**
+   * Returns the listen sockets of this server.
+   */
+  @NotNull
```

Comment:
This probably needs to go into the java doc rather than annotation

Reply:
fixed